plugin-server/src/cdp/_tests/fixtures.ts

```diff
@@ -15,11 +15,12 @@ import {
     IntegrationType,
 } from '../types'
 
-export const createHogFunction = (hogFunction: Partial<HogFunctionType>) => {
+export const createHogFunction = (hogFunction: Partial<HogFunctionType>, _status: string = 'active') => {
     const item: HogFunctionType = {
         id: randomUUID(),
         type: 'destination',
         name: 'Hog Function',
+        status: _status,
```

Comment:
Not sure what this is about? Firstly its the wrong thing (HogFunctions, not HogFlow) and the status is being added as a separate param to get around the missing value on the type (😬). I'm assuming this was just a mistake?